pele/gui/ui/mplwidget.py

```diff
@@ -1,6 +1,6 @@
 from PyQt4 import QtGui, QtCore
 from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
-from matplotlib.backends.backend_qt4agg import NavigationToolbar2QTAgg as NavigationToolbar
+from matplotlib.backends.backend_qt4agg import NavigationToolbar2QT as NavigationToolbar
```

Comment:
this because otherwise travis fails
On 3 Dec 2015 8:06 am, "Jacob Stevenson" <notifications@github.com> wrote:

> In pele/gui/ui/mplwidget.py
> <https://github.com/pele-python/pele/pull/138#discussion_r46522535>:
>
> > @@ -1,6 +1,6 @@
> >  from PyQt4 import QtGui, QtCore
> >  from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
> > -from matplotlib.backends.backend_qt4agg import NavigationToolbar2QTAgg as NavigationToolbar
> > +from matplotlib.backends.backend_qt4agg import NavigationToolbar2QT as NavigationToolbar
>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pele-python/pele/pull/138/files#r46522535>.
>